frontend/src/pages/home/aiFlows/AIFlowHint.tsx

```diff
@@ -0,0 +1,83 @@
+import * as React from 'react';
+import {
+  Button,
+  Card,
+  CardBody,
+  CardHeader,
+  Content,
+  Flex,
+  FlexItem,
+} from '@patternfly/react-core';
+import { TimesIcon } from '@patternfly/react-icons';
+import { useBrowserStorage } from '~/components/browserStorage';
+
+type AIFlowHintProps = {
+  title: React.ReactNode;
+  body: React.ReactNode;
+  isDisplayed: boolean;
+  image: React.ReactNode;
+};
```

Comment:
Just curious, will this component be reused and have different contents? If not, why not just put these values into the component?